Form/FormFlowInterface.php

```diff
@@ -178,4 +178,33 @@ function getSteps();
 	 */
 	function getStepCount();
 
+	/**
+	 * List steps for which there's data provided
```

Comment:
Steps that are done don't necessarily have submitted data. They could've been skipped as well. I guess a PHPDoc comment like

```php
/**
 * @return StepInterface[] Steps done.
 */
```

would suffice.